spec/fixtures/coconut-lib/Tests/CoconutTest.xib

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
is this file needed in order for the tests to pass?

Reply:
seems like it's not actually